debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -321,7 +323,7 @@ public static SnapshotMode parse(String value, String defaultValue) {
             .withType(Type.LIST)
             .withWidth(Width.LONG)
             .withImportance(Importance.LOW)
-            .withValidation(Field::isListOfRegex, MongoDbConnectorConfig::validateCollectionExcludeList)
+            .withValidation(Field::isListOfRegex, MongoDbConnectorConfig::validateCollectionBlacklist)
```

Comment:
This method is called for both legacy fields. Does it make sense to give a name related to that (`#validateCollectionLegacyField` or something)?

But it's only an aesthetic issue that `COLLECTION_WHITELIST` and `COLLECTION_BLACKIST` both call.`#validateCollectionBlacklist`.

Reply:
As earlier there was a single method doing the validation (ie validateExcludeList), so I thought to keep it likewise. 
Or maybe we can keep `validateCollectionIncludeExcludeField` and `validateCollectionWhitelistBlacklistField`